src/locales/hu.json

```diff
@@ -126,7 +126,7 @@
         "risk_tolerance_point1_percent_label": "Kockázatkerülő keret",
         "risk_tolerance_point1_percent_explanation": "Keret: évi 0.1% esély a fertőzésre, avagy 20 microCOVID hetente (javasolt érték, ha veszélyeztetett vagy, vagy gyakran találkozol veszélyeztetettel)",
         "intro": {
-            "whats_this": "<p>Ennek az oldalnak a segítségével meg tudod becsülni, hogy milyen eséllyel fertőződsz meg a koronavírussal különböző tevékenységek során. A <1>legjobb kutatási eredményeket</1> felhasználva hoztuk létre azzal a céllal, hogy segítsen használható intuíciót és megfelelő ártalomcsökkentési stratégiákat kialakítani, és csökkenteni a vírussal kapcsolatos stresszt.</p><p>A legfontosabb lépés a családod és a környezeted megóvására, ha te magad nem kapod el a koronavírust.</p>",
```

Comment:
@beshaya i think I remember @gyp saying that we shouldn't edit translation files directly. But I might be missing remembering. Can you double check with him?

Reply:
Yes, exactly, as documented in the README file next to the translations:
https://github.com/microcovid/microcovid/blob/main/src/locales/README-TRANSLATIONS.md

We should not be editing translation files directly as it wreaks all kinds of havoc in the process. Weblate will, in theory, pick up the changes and remove no longer needed translations.

Reply:
Removed